docs/gatsby-config.js

```diff
@@ -16,29 +16,43 @@ module.exports = {
           1: 'version-1',
         },
         sidebarCategories: {
-          null: ['index', 'getting-started', 'whats-new'],
-          Essentials: [
-            'essentials/schema',
-            'essentials/server',
-            'essentials/data',
+          null: [
+            'index',
+            'getting-started',
+            '[Changelog](https://github.com/apollographql/apollo-server/blob/master/CHANGELOG.md)',
           ],
-          Features: [
-            'features/caching',
-            'features/mocking',
-            'features/errors',
-            'features/data-sources',
-            'features/subscriptions',
-            'features/metrics',
-            'features/graphql-playground',
-            'features/scalars-enums',
-            'features/unions-interfaces',
-            'features/directives',
-            'features/creating-directives',
-            'features/authentication',
-            'features/testing',
-            'features/apq',
-            'features/health-checks',
-            'features/file-uploads',
+          'Defining a Schema': [
+            'defining-a-schema/schema',
+            'defining-a-schema/scalars-enums',
+            'defining-a-schema/unions-interfaces',
+            'defining-a-schema/directives',
+            'defining-a-schema/creating-directives',
+          ],
+          'Resolving Requests': [
+            'resolving-requests/data',
+            'resolving-requests/data-sources',
+            'resolving-requests/errors',
+            'resolving-requests/authentication',
+            'resolving-requests/file-uploads',
+            'resolving-requests/subscriptions',
+          ],
+          'Testing & Monitoring': [
```

Comment:
In my mind, _Monitoring_ is more closely coupled with _Deployment_ since it's more aligned with a production deployment story than _Testing_, which is more of a pre-production concern (read: development).